### PR TITLE
Midround pierats! Yar

### PR DESCRIPTION
**Pull request by Agoichi**
## About the PR

Made gamerule/midroundevent
Made MobHumanPirates
Made Spawners for them
Made a mid round pirates shuttle

There are three pirates on the ship
- Pirate
- Firstmate
- Captain
Their defoult weapon are pirate revolver and cutlas
Weapons On the ship:
- Stunprod
- Two pirate pistols
- One pirate shotgun
- One laser cutlass
- One svalin
## Why / Balance

We have pirates. They don't work properly. I want pirates.
We have some new brand shuttle guns added. Why not to make something to give them chance to show it's best?
Mid round pirates will help to make round more interesting. More to that mid pirates can'r live for long without station and they have everything to not act like a crazy guy with cannon but as a trader (who can cut open your throat after the trade)
## Media

https://github.com/space-wizards/space-station-14/assets/92464780/078e65d8-ed2b-4485-b6e2-e47494c626eb

https://github.com/space-wizards/space-station-14/assets/92464780/2a9ab0bb-575f-4365-bc44-41a0381bb8ad

https://github.com/space-wizards/space-station-14/assets/92464780/7349dd35-9ace-42ac-883b-c06a0418d704

Base Profile Screenshot 2024 01 23 - 22 33 23 74
Base Profile Screenshot 2024 01 23 - 22 33 44 98
Base Profile Screenshot 2024 01 23 - 22 33 14 89

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

LostPiratesSpawnRule was added

**Changelog**

:cl:
- add: Lost Pirates game rule
- add: Lost Pirates shuttle
- add: Lost Pirates ghost roles
- tweak: Changed Mobhuman pirates from npc to player
- tweak: Pirates now have headset with freelance channel

### CODE REVIEW

**Comment by Agoichi** [comment]
FUCK

**Comment by Agoichi** [comment]
Well.... give me some time for my comment to PR Lol

**Review by TheShuEd — CHANGES_REQUESTED** [review]
❤️

**Comment by TheShuEd** [comment]
transferring to draft, discussed with the author

**Review by Agoichi — COMMENTED** [review]
Done

**Comment by icekot8** [comment]
I don't like the shuttle at all. It's too powerful for midround antagonists, their shuttle will just get ripped off and make a huge profit :trollface:, also, it doesn't have a table for drinking rum... the horror.
Doesn't look like a pirate shuttle, it a converted syndicate operative's shuttle.

**Comment by Agoichi** [comment]
> I don't like the shuttle at all. It's too powerful for midround antagonists, their shuttle will just get ripped off and make a huge profit :trollface:, also, it doesn't have a table for drinking rum... the horror. Doesn't look like a pirate shuttle, it a converted syndicate operative's shuttle.

I disagree. Firstly, the have a table to drink rum. Secondly even if they have many weapons they have no bullets. They have one revolver on each guy, two pistols with one ammo, one shotgun with three ammo, one handed cannon (okay, this is maybe overpowered), one svalin and two shuttle canons with no additional bullets

**Comment by Agoichi** [comment]
Working on shuttle design a lil bit

**Comment by Agoichi** [comment]
Shuttle was re-designed
https://github.com/space-wizards/space-station-14/assets/92464780/229453c2-f000-4845-a1ba-c8d2a49cdf5a

File: Content.Server/StationEvents/Components/LostPiratesSpawnRuleComponent.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public sealed partial class LostPiratesSpawnRuleComponent : Component
+{
+    [DataField]
+    public string LostPiratesShuttlePath = "Maps/Shuttles/lost_pirates.yml";
+
+    [DataField]
+    public EntProtoId GameRuleProto = "Pirates";
+
+    [DataField]
+    public EntityUid? AdditionalRule;
```
Comment (metalgearsloth) [inline Content.Server/StationEvents/Components/LostPiratesSpawnRuleComponent.cs:17]: Why isn't this just using the pirates gamerule with a different prototype?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/StationEvents/Components/LostPiratesSpawnRuleComponent.cs (hunk 1 above)
Comment (Agoichi) [reply Content.Server/StationEvents/Components/LostPiratesSpawnRuleComponent.cs:17]: Well... PiratesGameRule don't work at all.....

**Comment by brainfood1183** [comment]
I like the overall shuttle shape but i dislike the interior design.

**Comment by Agoichi** [comment]
> I like the overall shuttle shape but i dislike the interior design. 

:')
Any suggestions?

**Comment by Agoichi** [comment]
Waiting for  #24490 guys

**Comment by metalgearsloth** [comment]
Whenever the actual antag selection rework is done https://github.com/space-wizards/space-station-14/pull/25786 open a new PR and update it because adding an entirely new gamerule to handle minor variations is just going to make antag code unmaintainable long-term.

**Pull request closed without merging**